package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mux.js",
-  "version": "2.3.0",
+  "version": "2.3.1",
```

Comment:
versioning is handled when we're ready for release, so, a PR shouldn't be updating it.